src/pocketdb/consensus/Helper.cpp

```diff
@@ -105,7 +106,9 @@ namespace PocketConsensus
             auto txType = PocketHelpers::TransactionHelper::ParseType(tx);
             if (txType == TxType::NOT_SUPPORTED)
                 continue;
-            if (coinstakeBlock && txType == TxType::TX_COINBASE)
+            // TAWMAZ: Coinbase exists in early mined blocks as well, 
+            // if (coinstakeBlock && txType == TxType::TX_COINBASE)
+            if (txType == TxType::TX_COINBASE)
                 continue;
```

Comment:
This is a bit different from COINBASE, the fact is that the zero transaction in any block is COINBASE, but in the case of PoS, it does not make any sense. The first 1020 blocks are a net issue for mining, then this transaction was left in the block (I think we can consider deleting it in the future and revise the algorithm for determining IsCoinStake)

At the same time, we must leave the transactions from the first 1020 blocks and check them further in this method

Reply:
The issue I'm running into here is the code is failing to create mined blocks in regtest for testing purposes.